server/Episememe.Application/Features/MediaFiltering/MediaFilter.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Episememe.Application.DataTransfer;
+using Episememe.Domain.Entities;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Episememe.Application.Features.MediaFiltering
+{
+    public class MediaFilter
+    {
+        private SearchMediaDto SearchMedia { get; }
```

Comment:
Because of CamelCase, it doesn't look like a private component of a class. Why not a `private readonly SearchMediaDto _searchMedia`?